cdk/domino_cdk/config/template.py

```diff
@@ -21,17 +21,16 @@ def config_template(
     destroy_on_destroy = False
     disk_size = 1000
     platform_instance_type = "m5.2xlarge"
-    platform_min_size = 3
+
     if dev_defaults:
         max_nodegroup_azs = 1
         destroy_on_destroy = True
         disk_size = 100
```

Comment:
do you still need this?

Reply:
In later lines you change disk_size=disk_size to disk_size=100

Reply:
Yes, I believe so? For "prod", we do want 1000G disks for compute & GPU and there isn't a way to set a variable optionally like that, no?

Reply:
I mean you have outer var disk_size and inner was disk_size in add_nodegroups. And you pass one into another. But add_nodegroups is already a closure that sees outer disk_size.

Reply:
Sorry for confusion, I'm talking about the different line, the definition of add_nodegroups().
It sees disk_size. Do you want to pass it?

Reply:
You have my approval anyway.

Just passing outer scope variables into the closure looks off to me